features/support/shell_overrides/ln

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# A test mock for "ln" (creates symlinks)
+
+echo "ln called with: $@"
```

Comment:
This is no longer needed. Also shouldn't you add shell overrides for mkdir and curl?